pkgs/by-name/kl/klassy/package.nix

```diff
@@ -0,0 +1,102 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  kdePackages,
+  libsForQt5,
+
+  # Could be either "5" or "6" for Qt 5 and 6 respectively.
+  qtMajorVersion ? "6",
```

Comment:
do we really need to support qt5 at this point?

Reply:
> do we really need to support qt5 at this point?

Depends on how many people still use Plasma 5 — in my experience, not a lot, but I could be wrong 🤷🏼‍♀️

Reply:
You can dispatch on `qtbase.version` here.